fixtures/aputure/c300d.json

```diff
@@ -0,0 +1,263 @@
+      }
+    },
+    "Mode Selection": {
+      "defaultValue": "0%",
+      "capabilities": [
+        {
+          "dmxRange": [0, 127],
+          "comment": "Manual Mode",
+          "type": "Maintenance"
+        },
+        {
+          "dmxRange": [128, 255],
+          "comment": "FX Mode",
+          "type": "Maintenance"
+        }
+      ]
+    },
+    "FX Control": {
+      "defaultValue": "0%",
+      "capabilities": [
+        {
+          "dmxRange": [0, 19],
+          "type": "Effect",
+          "effectName": "Paparazzi",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        },
+        {
+          "dmxRange": [20, 39],
+          "type": "Effect",
+          "effectName": "Fireworks",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        },
+        {
+          "dmxRange": [40, 59],
+          "type": "Effect",
+          "effectName": "Faulty bulb",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        },
+        {
+          "dmxRange": [60, 79],
+          "type": "Effect",
+          "effectName": "Lightning",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Lightning Mode Options"
+          }
+        },
+        {
+          "dmxRange": [80, 99],
+          "type": "Effect",
+          "effectName": "TV",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        },
+        {
+          "dmxRange": [100, 119],
+          "type": "Effect",
+          "effectName": "Pulsing",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        },
+        {
+          "dmxRange": [120, 139],
+          "type": "Effect",
+          "effectName": "Strobe",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        },
+        {
+          "dmxRange": [140, 159],
+          "type": "Effect",
+          "effectName": "Explosion",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Explosion Mode Options"
+          }
+        },
+        {
+          "dmxRange": [160, 255],
+          "type": "NoFunction",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        }
+      ]
+    },
+    "FX Frequency Control": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 19],
+          "comment": "1",
+          "helpWanted": "What speeds do these numbers represent?",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
```

Comment:
I would presume each range is a 'step' in speed. Currently, each range is a ramp from slow to fast, repeating, which is incorrect. How would one represent the desired 'stepping' behaviour?

Reply:
I'd probably set `"speed": "10%"`, `"speed": "20%"`, … `"speed": "100%"` to each capability. But combining them all into one with start/end would also be fine by me.